admin/class-yoast-dashboard-widget.php

```diff
@@ -38,6 +38,7 @@ public function __construct( WPSEO_Statistics $statistics = null ) {
 	 * Adds dashboard widget to WordPress
 	 */
 	public function add_dashboard_widget() {
+		add_filter( 'postbox_classes_dashboard_wpseo-dashboard-overview', array( $this, 'wpseo_dashboard_overview_class' ) );
```

Comment:
Haven't found a simpler way to change the default CSS classes used for dashboard widgets. This PR goal is to make "global" style changes easy, see the example made on the issue:
```
.yoast a {
    color: newcolor;
}
```

Not married to using this filter though 😄 we can leave out the dashboard widget, even if this would make the whole purpose of this PR a bit pointless.